cineast-core/src/main/java/org/vitrivr/cineast/core/db/DBSelectorSupplier.java

```diff
@@ -1,7 +0,0 @@
-package org.vitrivr.cineast.core.db;
-
-import java.util.function.Supplier;
-
-public interface DBSelectorSupplier extends Supplier<DBSelector> {
```

Comment:
This abstraction was introduced deliberately, since there is an `init()` method in both the `Extractor` as well as the `Retriever` which both take a `Supplier` with a different generic type and would therefore clash. Apparently, the `Extractor` now has a mandatory parameter `batchSize` which resolves that clash. This parameter does however appear to be ignored in many cases, so I don't think it should be mandatory if it isn't even consistently respected. Therefore, I suggest to re-introduce this abstraction in order to avoid the clash in generics.

Reply:
I resolved this and removed `batchSize` from the `init()` method. It is now a property of the `PersistencyWriter` itself.